docs/containerizedcomputation.md

```diff
@@ -0,0 +1,36 @@
+### Start the service:
+- Run `docker compose up funnel`
+- To try it with the rest of the stack, also run:
+`docker compose up htsgetDecrypt`
+
+
+### Access the service:
+- Go into the directory: `cd containerized-computation`
+- Get a token:
+    `token=$(curl -s -k https://localhost:8080/tokens | jq -r '.[0]')`
+- Put it in your task file:
+     `./fixtoken $token task1.json`
+- The task file `task1.json` contains instructions for downloading chromosome 11 of htsnexus_test_NA12878,
+  using htsget, and to count the number of lines in this file.
+- Submit your task: `curl 'localhost:8111/v1/tasks' --data @task1.json`
+- Open http://localhost:8111/ in your browser to see the status of the job.
+- When the job has completed, you should have a new file `linecount.txt` in your `output/`
+  folder. This file should contain the number of lines in the requested bam file.
+
+
+### About the task execution:
```

Comment:
Perhaps add a link to documentation for [GA4GH TESK](https://www.ga4gh.org/product/task-execution-service-tes/).